packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -241,6 +257,38 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
         move(_parentIndex, _claim, false);
     }
 
+    /// @inheritdoc IFaultDisputeGame
+    function addLocalData(uint256 _ident, uint256 _partOffset) external {
```

Comment:
On second thoughts, it's not unreasonable for the challenger to just retrieve the data from the L2OO itself rather than having to expose getters here.

Reply:
The idea was that we would expose getters for the data within the dispute game (`rootClaim`, `l2BlockNumber`, `l1Head`, etc.) and data like the output root can be fetched through the oracle or the local `op-node`'s `optimism_outputAtBlock` endpoint.